types/transactions.ts

```diff
@@ -910,7 +910,7 @@ export type PatchTransactionRequest = {
     transactionId: string
     
     data: {
-        type: "transaction" | "bookTransaction" | "chargebackTransaction"
+        type: "transaction"
         attributes: {
             summary?: string
```

Comment:
let's remove summary from this attributs and relationships from PatchTransactionRequest
no need for that, this request only for tags

Reply:
Done